src/cogs/bert.py

```diff
@@ -91,12 +91,31 @@ async def bert(self, ctx):
         if ctx.invoked_subcommand is None:
             await ctx.send_help(ctx.command)
 
+
+    @bert.command(name="top")
+    async def top(self, ctx, *content):
```

Comment:
add multi-line comment to method. these comments work as command descriptors and will show up in the help commands